### ISSUE TITLE
tldr -p and -u flags are not recognized but documented

### ISSUE THREAD

**Issue by alvinseville7cf** [tldr-pages/tldr#6248]
Hello! I installed tldr 0.4.0.1 via `sudo apt install tldr -y` on Ubuntu 20.04 LTS. Then I typed `tldr tldr` and obtained the following output:

```bash
tldr
Displays simple help pages for command-line tools, from the tldr-pages project.More information: https://tldr.sh.

 - Show the tldr page for a command (hint: this is how you got here!):
   tldr {{command}}

 - Show the tldr page for cd, overriding the default platform:
   tldr -p {{android|linux|osx|sunos|windows}} {{cd}}

 - Show the tldr page for a subcommand:
   tldr {{git-checkout}}

 - Update local pages (if the client supports caching):
   tldr -u
```
But when I tried execute smth like `tldr -p windows cd` or `tldr -u` error occurred:

```bash
Invalid option `-u'

Did you mean one of these?
    -h
    -v

Usage: tldr [-v|--version] [--update] COMMAND
  tldr Client program
```

Also man tldr tells nothing about -p and -u flags.
![Screenshot from 2021-07-19 17-08-35](https://user-images.githubusercontent.com/42812113/126117642-f578e96e-5331-45ee-88e7-255ded4a1883.png)

**Comment by CleanMachine1** [comment]
Hi, this topic has been raised many times.

To start the client you are using is very old. And doesn't have the options.
Here are some other pieces of info you should see


https://github.com/tldr-pages/tldr/issues/5962
https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#arguments
Just read beneath the table
https://github.com/tldr-pages/tldr/pull/6177
https://github.com/tldr-pages/tldr/pull/5965/files#r632621019

**Comment by CleanMachine1** [comment]
I personally would recommend the official python client (not tldr-py)

Since you have ubuntu, I think you have pip3 already installed, so just run `pip3 install tldr`

**Comment by CleanMachine1** [comment]
Feel free to close this issue, unless you are not satisfied.

However thanks for contributing, your effort is appreciated.
